src/util.cpp

```diff
@@ -226,6 +226,32 @@ ClassicLocale ClassicLocale::instance_;
 
 #endif  // !FLATBUFFERS_LOCALE_INDEPENDENT
 
+std::string EscapeAndWrapBuffer(const void *buffer, size_t buffer_size,  size_t max_length,
```

Comment:
This seems like it should be called `BufferToHexText` or something similar, to not be confused with the `EscapeString` function we already have. And if this code is only use by the binary schema dumper, just in-line it in that code rather than making it a general function here.

Reply:
Ok, let me fix up this function name.